tests/stories/Example.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="ts">
+  import type { Snippet } from 'svelte';
   import type { HTMLButtonAttributes } from 'svelte/elements';
 
-  interface Props extends HTMLButtonAttributes {
+  type Props = {
     rounded?: boolean;
-  }
+    children?: Snippet;
+  } & Pick<HTMLButtonAttributes, 'onclick' | 'onmouseenter' | 'onmouseleave'>;
```

Comment:
Nit: This probably can be simplified to this:

```suggestion
  } & Pick<HTMLButtonAttributes, 'children' | 'onclick' | 'onmouseenter' | 'onmouseleave'>;
```